extensions/database-documentation/src/extension.ts

```diff
@@ -0,0 +1,49 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+// The module 'azdata' contains the ADS extensibility API
+import * as azdata from 'azdata';
+
+// The module 'vscode' contains the VS Code extensibility API
+// Import the module and reference it with the alias vscode in your code below
+import * as vscode from 'vscode';
+
+// The module 'azdata' contains the Azure Data Studio extensibility API
+// This is a complementary set of APIs that add SQL / Data-specific functionality to the app
+// Import the module and reference it with the alias azdata in your code below
+import { generateMarkdown } from './common/utils';
+
+// The module 'openai' contains the OpenAI API
+import { Configuration, OpenAIApi } from "openai";
+
+
+// this method is called when your extension is activated
+// your extension is activated the very first time the command is executed
+export async function activate(context: vscode.ExtensionContext) {
+
+    // Use the console to output diagnostic information (console.log) and errors (console.error)
+    // This line of code will only be executed once when your extension is activated
+    console.log('Congratulations, your extension "database-documentation" is now active!');
+
+
+    // The command has been defined in the package.json file
+    // Now provide the implementation of the command with  registerCommand
+    // The commandId parameter must match the command field in package.json
+    context.subscriptions.push(vscode.commands.registerCommand('database-documentation.generateDocumentation', async (context: azdata.ObjectExplorerContext) => {
+        // The code you place here will be executed every time your command is executed
+        vscode.window.showInformationMessage("Generating Documentation... ");
```

Comment:
This (and all other user-visible messages) should be localized (see other extensions for an example using the `localize` function)